truffle/LoadContract.html

```diff
@@ -157,15 +169,51 @@
 end
 
 loop While in Progress
-C ->> L: setVaultHash()
-activate L
-L -->> SL: setVaultHash(hash)
-activate SL
-deactivate SL
-note over L: emit VaultHash
-deactivate L
+C ->> V: setVaultHash(vaultId, vaultHash)
+activate V
+note over V: emit VaultHash
+deactivate V
 end
 
+opt Need to Revoke the Permission of Carrier
+S->>V: revokeUpdateHashPermission(vaultID, Carrier)
+activate V
+deactivate V
+note over V: UpdateHashPermissionRevoked
+end
+
+
+
+opt Need to Grant the Hash Permission of One Participant
```

Comment:
I think it looks better visually if we group all of the Acl updates into one group of optional commands - since the ACL updates are independent of the shipment lifecycle. This includes the grantUpdateHashPermission(vaultId, Carrier) above.

You can just have generic addressToGrant/addressToRevoke examples, that can only be invoked by `[ Shipper ]`